runtime/Cargo.toml

```diff
@@ -58,3 +58,4 @@ fips_self_test=[]
 no-cfi = ["caliptra-image-verify/no-cfi", "caliptra-drivers/no-cfi"]
 fpga_realtime = ["caliptra-drivers/fpga_realtime"]
 "hw-1.0" = ["caliptra-builder/hw-1.0", "caliptra-drivers/hw-1.0", "caliptra-registers/hw-1.0", "caliptra-kat/hw-1.0","caliptra-cpu/hw-1.0"]
+arbitrary_max_handles = ["dpe/arbitrary_max_handles"]
```

Comment:
I don't feel too strongly about this, but we probably don't need to define a feature on the RT crate for this. On the DPE dependency we can just add `features = ["arbitrary_max_handles"]`